### PR TITLE
Updated convolution kernel to work with embARC MLI Library 2.0 for ARC

### PR DESCRIPTION
**Pull request by gerbauz**
[1/11]

This PR brings updates for ARC convolution kernel to support [embARC MLI Library 2.0](https://github.com/foss-for-synopsys-dwc-arc-processors/embarc_mli/tree/Release_2.0_EA) as an experimental feature.

BUG=#86

### CODE REVIEW

File: tensorflow/lite/micro/kernels/arc_mli/conv.cc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+/* Copyright 2020-2021 The TensorFlow Authors. All Rights Reserved.
```
Comment (advaitjain) [inline tensorflow/lite/micro/kernels/arc_mli/conv.cc:1]: our license check only accepts a single year. Please update to be `Copyright 2021 ...`

**Review by advaitjain — APPROVED** [review]

**Comment by advaitjain** [comment]
The only bit that remains for the current PR is that the branch is out of date with the base branch.

For most PRs, I see a button that allows me to merge main into the PR branch, which looks something like this:
image

For this particular PR, I do not see the update branch button (only the message that the branch is out-of-date). My guess is that the option to allow maintainers to modify the PR was unchecked while creating this PR.

**Comment by gerbauz** [comment]
> The only bit that remains for the current PR is that the branch is out of date with the base branch.
> 
> For most PRs, I see a button that allows me to merge main into the PR branch, which looks something like this:
> image
> 
> For this particular PR, I do not see the update branch button (only the message that the branch is out-of-date). My guess is that the option to allow maintainers to modify the PR was unchecked while creating this PR.

OK, I'll update all PRs of course.

**Comment by gerbauz** [comment]
Hi @advaitjain, for some reason I can't find this checker you were talking about. Also, I've asked another person - he also doesn't have this checker. That's strange, but I have "update branch" button and I'll click it on all ARC pull requests so you can merge them (if you don't mind).

**Review by advaitjain — APPROVED** [review]

**Comment by advaitjain** [comment]
> Hi @advaitjain, for some reason I can't find this checker you were talking about. Also, I've asked another person - he also doesn't have this checker. That's strange, but I have "update branch" button and I'll click it on all ARC pull requests so you can merge them (if you don't mind).

I think what you're hitting is https://github.com/isaacs/github/issues/1681

It's going to be very painful to have you update the PRs. If at all possible, u would strongly suggest sending the PR from a personal fork of tflite-micro.

**Comment by gerbauz** [comment]
> > Hi @advaitjain, for some reason I can't find this checker you were talking about. Also, I've asked another person - he also doesn't have this checker. That's strange, but I have "update branch" button and I'll click it on all ARC pull requests so you can merge them (if you don't mind).
> 
> I think what you're hitting is [isaacs/github#1681](https://github.com/isaacs/github/issues/1681)
> 
> It's going to be very painful to have you update the PRs. If at all possible, u would strongly suggest sending the PR from a personal fork of tflite-micro.

I see, but unfortunately I can't do it right now. Can I just manually update all of them one by one?

**Comment by advaitjain** [comment]
ok, let's have you update them one at a time for now.

**Pull request merged**